composer.json

```diff
@@ -120,7 +117,18 @@
     "prefer-stable": true,
     "config": {
         "discard-changes": true,
-        "sort-packages": true
+        "allow-plugins": {
+            "composer/installers": true,
+            "cweagans/composer-patches": true,
+            "drupal/core-composer-scaffold": true,
+            "drupal/core-project-message": true,
+            "dealerdirect/phpcodesniffer-composer-installer": true
+        },
+        "preferred-install": {
+            "archipelago/*": "source",
+            "strawberryfield/*": "source"
+            "*": "dist"
```

Comment:
```suggestion
            "strawberryfield/*": "source",
            "*": "dist"
```

Reply:
Just a missing comma.

Reply:
Thanks! fixed. Let me know if you see anything else.